### DISCUSSION TITLE
Pausing a running script?

### DISCUSSION THREAD

**Discussion by crites** [Stellarium/stellarium#1859] [Q&A]
Can anyone tell me how to pause and restart a running script? 

I've seen some pretty interesting scripts that tour the heavens but they aren't much use when they zip through all the objects and you can't keep up.

**Comment by alex-w** [comment]
Did you tried use GUI of shortcuts to manager script running?

**Reply by adgorn** [reply]
I'm running the Binocular Highlights script.  The Crtl D S won't stop it.  How do I stop it and exit?  I can't see the interface anymore.

**Comment by gzotti** [comment]
As we have written in the User Guide of 1.0 and later, one difference between the Qt5 line and Qt6 line of Stellarium is the non-availability of script pausing in the latter. This is out of our hands.

**Comment by adgorn** [comment]
Got it. Windows Task Manager>End Task works for this. Thank you. 

Sent from AT&T Yahoo Mail on Android 
 
  On Mon, Jan 9, 2023 at 12:11 PM, Georg ***@***.***> wrote:   


As we have written in the User Guide of 1.0 and later, one difference between the Qt5 line and Qt6 line of Stellarium is the non-availability of script pausing in the latter. This is out of our hands.

**Reply by alex-w** [reply]
Hmm... Ctrl+T & Ctrl+Q

**Comment by adgorn** [comment]
Indeed!
    On Tuesday, January 10, 2023 at 01:59:02 PM CST, Alexander V. Wolf ***@***.***> wrote:  
 
 


Hmm... Ctrl+T & Ctrl+Q
